config.json

```diff
@@ -7,7 +7,7 @@
     // Mode in which the dashboard server express environment runs
     NODE_ENV: development
     // Interface on which the dashboard server listens
-    IP: 127.0.0.1
+    IP: 0.0.0.0
```

Comment:
Definitely want to keep 127.0.0.1 as the default so the server comes up on localhost only unless the user purposely overrides it (like we do in the Dockerfile.server).